portfolio/src/main/webapp/styles/sass/project-detail-page.scss

```diff
@@ -0,0 +1,13 @@
+main#project-detail {
+    & > h1 {
+        font-size: 48px;
+        font-family: Montserrat;
+        color: var(--text-color);
+    }
+
+    & > #description {
+        font-size: 18px;
+        font-family: Montserrat;
+        color: var(--text-color);
+    }
+}
```

Comment:
Nit: add linebreaks and the end of files so that the github UI doesn't complain :)

Reply:
Added.